requirements/base.in

```diff
@@ -1,5 +1,6 @@
 appdirs>=1.4
 attrs>=19.1.0
+cookiecutter~=2.1
```

Comment:
I debated adding this as an optional dependency in the `extras` to avoid pulling in all these extra dependencies just for a single `taskgraph init` command.

I decided against doing that because `taskgraph init` is supposed to be beginner friendly, and not everyone running it will be familiar enough with Python / pip to know how to install extras. But I can definitely be persuaded to omit it from the main dependencies if people feel strongly about it.

Reply:
This seems pretty sensible to me.